jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -205,6 +227,29 @@ private static String getCharsetName(String primary, Charset defaultCharset) {
 		}
 		return null;
 	}
+
+	/*
+	 * Return the appropriate System console using the pre-validated encoding.
+	 */
+	static PrintStream createConsole(FileDescriptor desc, String encoding) {
```

Comment:
I think this can be `private` - it's only used in this class.

Reply:
This doesn't seem to have been addressed.
Also, `createConsole(FileDescriptor)` below can also be `private`.

Reply:
I misspoke: `createConsole(FileDescriptor)` is private, but the modifiers are not in canonical order (it should be `private static ` instead of `static private`).

Reply:
Fixed.